cli/workspace/workspace.go

```diff
@@ -79,16 +89,53 @@ func CreateWorkspaceFileIfNotExists() (string, string, error) {
 	if err != nil {
 		return "", "", err
 	}
-	if _, err := f.WriteString(`workspace(name = "` + filepath.Base(workspacePath) + `")` + "\n"); err != nil {
+	contents := ""
+	if useModules {
+		contents = `module(name = "` + filepath.Base(workspacePath) + `")` + "\n"
+	} else {
+		contents = `workspace(name = "` + filepath.Base(workspacePath) + `")` + "\n"
+	}
+	if _, err := f.WriteString(contents); err != nil {
 		return "", "", err
 	}
 	pathVal = workspacePath
 	basename = fileName
 	pathErr = nil
-	log.Debugf("Created workspace file at %s/%s", pathVal, basename)
+	log.Debugf("Created %s file at %s/%s", fileName, pathVal, basename)
 	return pathVal, basename, nil
 }
 
+func useModules() bool {
+	data, err := os.ReadFile(".bazelversion")
+	if err != nil {
+		return true
+	}
+
```

Comment:
we have `bazelisk.ResolveVersion`, would that work here?

Reply:
I tried that at first, but it caused a circular dependency (because `bazelisk.go` depends on `workspace.go`).

I'd like to nuke the whole WORKSPACE generation bit at some point when I'm more confident with the module side of things - so I think this is probably alright for now.